src/test/libcephd/CMakeLists.txt

```diff
@@ -16,7 +16,7 @@ add_executable(ceph_test_cephd_api_misc
 set_target_properties(ceph_test_cephd_api_misc PROPERTIES COMPILE_FLAGS
   ${UNITTEST_CXX_FLAGS})
 target_link_libraries(ceph_test_cephd_api_misc
-  cephd global ${UNITTEST_LIBS} cephdtest z snappy ceph_zstd)
+	cephd global ${UNITTEST_LIBS} cephdtest z snappy ceph_zstd ${SSL_LIBRARIES})
```

Comment:
maybe we should add this linkage to `radosgw_a` instead. because it is radosgw_a which includes `civetweb_common_objs`, and the later has `civetweb.o`. so i'd recommend
```diff
diff --git a/src/rgw/CMakeLists.txt b/src/rgw/CMakeLists.txt
index 4a06cf3..d9a2ca7 100644
--- a/src/rgw/CMakeLists.txt
+++ b/src/rgw/CMakeLists.txt
@@ -139,7 +139,7 @@ endif (WITH_RADOSGW_ASIO_FRONTEND)

 add_library(radosgw_a STATIC ${radosgw_srcs}
   $<TARGET_OBJECTS:civetweb_common_objs>)
-target_link_libraries(radosgw_a rgw_a)
+target_link_libraries(radosgw_a rgw_a ${SSL_LIBRARIES})

 add_executable(radosgw rgw_main.cc)
 target_link_libraries(radosgw radosgw_a librados
@@ -147,7 +147,7 @@ target_link_libraries(radosgw radosgw_a librados
   cls_log_client cls_statelog_client cls_timeindex_client
   cls_version_client cls_replica_log_client cls_user_client
   global ${FCGI_LIBRARY} ${LIB_RESOLV}
-  ${CURL_LIBRARIES} ${EXPAT_LIBRARIES} ${SSL_LIBRARIES} ${BLKID_LIBRARIES}
+  ${CURL_LIBRARIES} ${EXPAT_LIBRARIES} ${BLKID_LIBRARIES}
   ${ALLOC_LIBS})
 # radosgw depends on cls libraries at runtime, but not as link dependencies
 add_dependencies(radosgw cls_rgw cls_lock cls_refcount
```
instead.